ckcp/openshift_dev_setup.sh

```diff
@@ -272,6 +272,12 @@ patches:
         description: This value refers to the hostAddress defined in the Route.
         value: $ckcp_route " >>"$ckcp_temp_dir/kustomization.yaml"
 
+	# Enable the "runtime/default" seccomp profile (required for restricted security context)
+	# This is enabled by default in restricted-v2 available in openshift >= 4.11
+	if ! (set +o pipefail && kubectl describe scc restricted restricted-v2 2>/dev/null |
```

Comment:
We are patching only restricted in the next line. Why are we describing restricted-v2 as well?

Reply:
This PR enables "runtime/default" seccomp profile in Pod's securityContext.
Openshift version < 4.11 complains if we use this seccomp profile because it is not enabled by default in the available SCCs, thus we are patching the restricted SCC here to support this seccomp profile.
The condition where we don't need to make this scc patch:
- It was already done (describing scc restricted)
- It is already available in openshift >= 4.11 (describing restricted-v2 scc) so no need to patch scc restricted.